vars/FreeBSD.yml

```diff
@@ -1,3 +1,11 @@
+__argus_radium_user: argus
+__argus_radium_user_home: /var/log/argus
+__argus_radium_user_shell: /usr/sbin/nologin
+__argus_radium_user_uid: 561
```

Comment:
not specific to the role, but here is a question: if UID, or GID, is not managed by a distribution, what should we do?

when a package creates users and groups, and manages UID and GID according to distribution's rule, everything is just fine. but sometimes not.

case 1: FreeBSD's `argus-clients` does not create a user. adding a user without UID generates a UID in 1000~. I do not think that it would cause serious issues in real life situation, but it is confusing when most FreeBSD users assume UID 1000~ is reserved for regular users, and ~1000 for daemons. you may create a user, or a group, with UID/GID less than 1000 by managing the numbers in the role, but it is also confusing because the range is reserved by `/usr/ports/(UIDs|GIDs)`.

case 2: Red Hat variants seem to create users and groups with whatever UID/GID available at the creation time. Even users created by packages.

in either cases, when you manage UIDs/GIDs, it is very likely that the number collides, especially when the number is more than 1000.

my solution, used in `ansible-role-argus`, is that; do not bother. just create a user and a group without specific ID. it looks messy but it does not conflict with user-defined, or distribution-defined IDs.

whan do you think?